commands/dumpthread.ts

```diff
@@ -0,0 +1,46 @@
+import { Message, ThreadChannel } from "discord.js";
+import { extractTrack, extractUrl, matchOrder, parseSubmission } from "../utils/gramophone";
+
+export default {
```

Comment:
The reason I have the JankbotCmd stuff is so extra checks can be done before running i.e. restricting to mods, and allow for sorting into folders. Might be worth making it a mod command.